recipes/server_check_helper.rb

```diff
@@ -0,0 +1,23 @@
+
+require 'ipaddr'
+
+def is_rogue_server?(retry_join_list, my_ip)
```

Comment:
Just a suggestion, take it or leave it. Maybe the name should be descriptive of what it's doing not the PMAI?

Like `is_server_disallowed?` or invert the logic with a different name.